README.md

```diff
@@ -5,17 +5,11 @@ This chart is intended for adding the postgres sql databases.
 ## Prerequisites
 
 - To use this chart, you need a postgres Flexible server created beforehand.
-- Please see [this](https://github.com/hmcts/cnp-flux-config/blob/master/apps/sscs/preview/aso/sscs-postgres.yaml) link where the Flexible server is deployed using ASO.
+- Please see [cnp-flux-config](https://github.com/hmcts/cnp-flux-config/pull/24968) PR link where the Flexible server is deployed using ASO.
 
 ## Example configuration
 
-```yaml
```

Comment:
there should still be an example here I think you can restore it and leave the PR as well

Reply:
sure - done